docs/setup-and-configuration.rst

```diff
@@ -287,6 +287,7 @@ Plugin settings:
 Storage settings:
 
 
+* ``max_upload_size`` (integer) - Maximal upload size in bytes. Keep in mind that this value refers to whole upload request (``Content-Lenght`` from request header), so the maximal file size is smaller than that by +/- 500B. Default is ``None``, which means there is no limit.
```

Comment:
```suggestion
* ``max_upload_size`` (integer) - Maximum upload size in bytes. Keep in mind that this value refers to whole upload request (``Content-Length`` from request header), so the maximum file size is smaller than that by +/- 500B (because of additional payload with metadata). Default is ``None``, which means there is no limit.
```